android/app/src/main/res/values/colors.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="launch_black">#131416</color>
```

Comment:
This appears to predate the newer color (`?android:colorBackground`) being used for API 31+, so I removed it